doc/man3/OPENSSL_malloc.pod

```diff
@@ -232,16 +227,19 @@ always because allocations have already happened).
 CRYPTO_mem_ctrl() returns -1 if an error occurred, otherwise the
 previous value of the mode.
 
-OPENSSL_mem_debug_push() and OPENSSL_mem_debug_pop()
-return 1 on success or 0 on failure.
-
 =head1 NOTES
 
 While it's permitted to swap out only a few and not all the functions
 with CRYPTO_set_mem_functions(), it's recommended to swap them all out
-at once.  I<This applies specially if OpenSSL was built with the
-configuration option> C<crypto-mdebug> I<enabled.  In case, swapping out
-only, say, the malloc() implementation is outright dangerous.>
+at once, especially if OpenSSL was built with the
+configuration option> C<crypto-mdebug>.
+
+=head1 HISTORY
+
+OPENSSL_mem_debug_push(), OPENSSL_mem_debug_pop(),
+CRYPTO_mem_debug_push(), and CRYPTO_mem_debug_pop()
+were deprecated in OpenSSL 3.0.0.
```

Comment:
3.0

Reply:
You sure?  Do ```git grep 3.0.0``` in the man directory.  (I did such a grep before changing the file.)

Reply:
Others have told me this when I used "3.0.0". It's quite possible that we haven't been consistent, though.

Reply:
```
; git grep --fixed 3.0 | grep -v 3.0.0 | wc
     29     241    2316
; git grep --fixed 3.0.0 | wc
     32     232    2492
```

Reply:
Sounds like the project should pick one, update doc-nits, and include that in a PR to fix the "wrong ones."  But not for this PR.

Reply:
It should be 3.0, but making all occurrences consistent is a task for a separate pr imho.

Reply:
Ask, and you shall be served.  #9376

Reply:
But what about
```
; git grep --fixed 1.1.0 | wc -l
191
; git grep --fixed 1.0.0 | wc -l
54
```
?

Reply:
Different version scheme, those *do* make sense.

Reply:
Anyway, #9376 is now merged, so...